### DISCUSSION TITLE
As a LibreChat admin running an instance for close family and friends, I'd like an invite-based registration system.

### DISCUSSION THREAD

**Discussion by th3raid0r** [danny-avila/LibreChat#2682] [Feature Requests & Suggestions]
Currently, I can get around this by using Google Auth and adding each users' gmail account as a test account. I could also likely leverage a cloud provider auth to perform this as well. However, both options are a bit much for my use case - especially since many of my users are non-technical to the extreme and sometimes lack a gmail account entirely. 

Instead of the above, I'd like a simple admin UI to "invite" people with only an email - leveraging the existing SMTP provider configuration. The user should then simply be able to click a link/button in the email, and be prompted to setup a password OR link with a supported social login.

**Comment by danny-avila** [comment]
Something like this is in the works, thanks for the suggestion!
